src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -242,7 +251,7 @@ def post(self, request):
                     self.open_resolve_dialog(data, group, integration)
                     defer_attachment_update = True
         except client.ApiError as e:
-            return self.api_error(e)
+            return self.api_error(e, action_type, logging_data)
```

Comment:
Hmmm, will we always have `action_type` here?

Reply:
@manuzope There should be because we should never have an `client.ApiError` unless we've gone through at least one action. But I think it's safer to initialize `action_type` to `None` just to be more safe.

Reply:
I'd put `action_type = None` before the `try` so that we know we always have it.